inc/core/templates/theme-notification-banner.php

```diff
@@ -0,0 +1,315 @@
+												value="1" 
+												<?php
+                                                if ( $notice_enabled ) {
+													echo ' checked="checked"';
+												}
+                                                ?>
+												/>
+										<span class="message">Enable / disable banner</span>
+									</label>
+								</div>
+							</div>
+						</div>
+						<div class="field">
+							<div class="label">
+								<label for="notice_homepage_only">Homepage only</label>
+							</div>
+							<div class="input">
+								<div class="true-false">
+									<label>
+											<input 	type="checkbox" 
+													id="notice_homepage_only" 				
+													name="notice_homepage_only" 
+													value="1" 
+													<?php
+                                                    if ( $notice_homepage_only ) {
+														echo ' checked="checked"';
+													}
+                                                    ?>
+													/>
+											<span class="message">Show only on the homepage</span>
+									</label>
+								</div>
+							</div>
+						</div>
+						<div class="field">
+							<div class="label">
+								<label for="notice_status">Status (colour)</label>
+							</div>
+							<div class="input">
+								<select id="notice_status" class="" name="notice_status">
+									<option value="notice-default" 
+									<?php
+									if ( 'notice-default' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Default (BCGov blue)</option>
+									<option value="notice-green" 
+									<?php
+									if ( 'notice-green' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Notice (green)</option>
+									<option value="notice-white" 
+									<?php
+									if ( 'notice-white' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Minimalist (white)</option>
+									<option value="notice-black" 
+									<?php
+									if ( 'notice-black' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Cancelled (dark)</option>
+									<option value="notice-yellow" 
+									<?php
+									if ( 'notice-yellow' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Warning (yellow)</option>
+									<option value="notice-orange" 
+									<?php
+									if ( 'notice-orange' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Alert (orange)</option>
+									<option value="notice-red" 
+									<?php
+									if ( 'notice-red' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Emergency (red)</option>
```

Comment:
These repeated options might be a good candidate for a foreach loop. Say you have the slug and human-readable colours as key and value(respectively) in an associative array. Something to consider if you ever add/modify this.